guides/source/debugging_rails_applications.md

```diff
@@ -849,6 +800,63 @@ set forcestep
 set listsize 25
 ```
 
+Debugging with the `web-console` gem
+------------------------------------
+
+Web Console is a bit like `byebug`, but it runs in the browser. In any page you
+are developing, you can request a console in the context of a view or a
+controller. The console would be rendered next to your HTML content.
+
+### Console
+
+Inside any controller action or view, you can then invoke the console by
+calling the `console` method.
+
+For example, in a controller:
+
+```ruby
+class PostController < ApplicationController
+  def new
+    console
+    @post = Post.new
+  end
+end
+```
+
+Or in a view:
+
+```html+erb
+<% console %>
+
+<h2>New Post</h2>
+```
+
+This will render a console in the browser on top of the post creation HTML
+view. You don't need to care for the `console` output, the console won't be
+rendered on the spot of the `console` invocation.
+
+The console executes pure Ruby code. You can define and instantiate
+custom classes, create new models and inspect variables.
+
+NOTE: Only one console can be rendered per request. Newer versions of
```

Comment:
I guess you can simply write:

> NOTE: Only one console can be rendered per request. Otherwise, `web-console` will raise an error on other `console` invocations.

"Newer" will most likely be wrong in the near future and it's important to document the current behavior, not when it was added. What do you think ?